docs/docs/tooling/profiler.md

```diff
@@ -25,32 +41,32 @@ fn main(ptr: pub u32, mut array: [u32; 32]) -> pub [u32; 32] {
     array
 }
 ```
-You can use these values for the `Prover.toml`:
-```toml
-ptr = 1
-array = [1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1]
-```
 
-Running `nargo info` we can get some information about the opcodes produced by this program, but it doesn't give us a lot of info on its own. Compile and execute this program normally using `nargo compile` and `nargo execute`.
+Change directory into the project and compile the program using `nargo compile`. We are ready then to try the profiler out.
 
-### Generating an ACIR opcode flamegraph
+#### Flamegraphing
 
-The program on its own is quite high-level. Let's get a more granular look at what is happening by using `noir-profiler`.
+Let's take a granular look at our program's ACIR opcode footprint using the profiler, running:
 
-After compiling the program, run the following:
 ```sh
 noir-profiler opcodes --artifact-path ./target/program.json --output ./target/
 ```
-Below you can see an example flamegraph with a total 387 opcodes (using `nargo` version 1.0.0-beta.2):
+
+The command will generate a flamegraph in your _target_ folder that maps the number of ACIR opcodes and their corresponding locations in your program source code.
+
+Opening the flamegraph in a web browser will provide a more interactive experience, allowing you to click into different regions of the graph and examine them.
+
+Flamegraph of the demonstrative project generated with Nargo v1.0.0-beta.2:
+
 ![ACIR Flamegraph Unoptimized](@site/static/img/tooling/profiler/acir-flamegraph-unoptimized.png)
 
-You should now have a flamegraph that maps ACIR opcodes to their corresponding locations in the source code. We strongly recommend generating these graphs yourself as you follow this guide. Opening the flamegraph in a browser provides a more interactive experience, allowing you to click into and examine different regions of the graph. Simply viewing the image file won't offer the same level of insight.
+The demonstrative project consists of 387 ACIR opcodes in total, which from the flamegraph we can see that the majority of them comes from the write to `array[i]`.
```

Comment:
```suggestion
The demonstrative project consists of 387 ACIR opcodes in total. From the flamegraph, we can see that the majority come from the write to `array[i]`.
```
I see you want to add info about the total projects. I think this sentence becomes clearer when split.